packages/css/src/base.css

```diff
@@ -37,6 +39,22 @@ body,
   background: var(--ds-color-neutral-background-default);
 }
 
+.ds-readonly-icon::before {
```

Comment:
Should we add a changeset that we have added this class? Could it be useful for our users to compose the icon for their own readonly elements? 🤔

Reply:
I think yes, we also need documentation for these classnames - such as `ds-focus`, etc.

Reply:
:) will you guys add this?

Reply:
Can do :)

Reply:
```md
---
"@digdir/designsystemet-css": patch
---

Added new class `ds-readonly-icon` for displaying a the readonly icon (padlock) before an element.
```

Reply:
hehe, usikker på hva jeg skal gjøre nå - så Tobias skrev "we also need documentation for these classnames" - det var det dere gjør? Bare gjør det dere ønsker med denne branchen, viktigste for oss er å få det inn :D

Reply:
Vi tar det del av #2207